Include/internal/pycore_gc.h

```diff
@@ -329,6 +334,7 @@ struct _gc_runtime_state {
     Py_ssize_t work_to_do;
     /* Which of the old spaces is the visited space */
     int visited_space;
+    int phase;
```

Comment:
Maybe this can be rolled into the `collecting` field (like 0 for not collecting, 1 for MARK, 2 for COLLECT)?

Reply:
Perhaps, but not very cleanly.
The phase depends on where we are in a full heap scavenge. The collecting flag will flip many times during a scavenge, for incremental collections.